global.json

```diff
@@ -1,5 +1,6 @@
 {
     "projects": [
-        "src"
+        "src",
+        "tools"
```

Comment:
I just checked. This doesn't cause a new package to get created. So you can disregard the above comment.